### ISSUE TITLE
reduce background image resolution option for full screen mode?

### ISSUE THREAD

**Issue by ben401rs** [JosefNemec/Playnite#3513]
### Feature description

It slows down my old laptop which i used solely for emulation, no matter which themes, if i used the fullscreen mode with the background images enabled, it loads slowly waiting for each background image to load. For some reason the desktop mode does have a lower image resolution options, and that one doesn't slow down Playnite for the background image to load.

### Screenshots

![image](https://github.com/JosefNemec/Playnite/assets/99275631/2853d6f9-8b82-4a1a-9bd9-a5d5c52c493d)

The option i mentioned

**Comment by BenjiBenji69** [comment]
https://youtu.be/94zhM_3ThmI

https://youtu.be/h0NSDivy38A

**Comment by BenjiBenji69** [comment]
i know it could be that, but it also can be caused by the fullscreen mode background always loads the highest quality one, i tried the desktop one that has heavy theme background that slows Platnite down on that laptop too, it loads the same as that but if i tick the option to lower it in the option, it magically doesnt slow down selections and menu anymore.

**Comment by JosefNemec** [comment]
Again, both modes load images in the exact same way unless a theme changes it. Those option from your screenshot only affect blur effect, they have nothing to do with how an image is actually loaded. 

The actual image rescaler can be changed in the settings (general section in FS mode, advanced -> performance in Desktop mode). But again, custom themes can ignore this and load images differently.

So unless you can replicate this with the default theme, you need to report this to theme dev.

**Comment by BenjiBenji69** [comment]
maybe im a bit vague, im only refering that tiny options for highest quality in desktop mode appearance settings, it seems the fullscreen mode doesn't have that in the option. And if i tick that also in any desktop themes, it slows the hell down of any selections in the menu, because it loads the highest quality background images to load to me at least

**Comment by JosefNemec** [comment]
That "high quality" option is for using high quality blurring algorithm.

**Comment by BenjiBenji69** [comment]
yeah i suppose that. The blurring slows down the whole program in fullscreen mode, maybe im not familiar with how devs made the fullscreen theme, but some does run fine with background images enabled mind you, usually it does loads fast for selection, maybe you can replicate it using some old laptop with integrated gpu, because its hard for me to exactly tell whats the problem and im running just fine without it, but it does slow down the whole program for me

**Comment by JosefNemec** [comment]
Fullscreen mode uses high quality blur always (if you have even small amount of blur enabled). No idea why there's no quality toggle in FS mode, must be an oversight. I could add blur quality toggle into FS mode in future.

Integrated GPUs are known to have issue with how blurring effect is done by UI library we use, nothing you can really do about it other than disabling blur completely.

**Comment by BenjiBenji69** [comment]
ok, good to know, for reference i did try every single desktop themes, heavier ones to be exact, and it runs slow on any selections when i enabled that high quality option. Even just clicking a library games slow massively waiting for the image or the blurring to loads, It doesn't slow down anymore if i untick it, and the background still shows as usual just lower quality ( to be honest i dont see whats the difference). Maybe the fullscreen one have that option enabled by default, or maybe other reasons i dont know
